src/Tribe/CSV_Importer/RSVP_Importer.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+
+/**
+ * Class Tribe__Tickets__CSV_Importer__RSVP_Importer
+ */
+class Tribe__Tickets__CSV_Importer__RSVP_Importer extends Tribe__Events__Importer__File_Importer {
+
+	/**
+	 * @var array
+	 */
+	protected $required_fields = array( 'event_name', 'ticket_name' );
+
+	/**
+	 * @var array
+	 */
+	protected static $event_name_cache = array();
+
+	/**
+	 * @var array
+	 */
+	protected static $ticket_name_cache = array();
+
+	/**
+	 * @var Tribe__Tickets__RSVP
+	 */
+	protected $rsvp_tickets;
+
+	/**
+	 * @var bool|string
+	 */
+	protected $row_message = false;
+
+	/**
+	 * The class constructor proxy method.
+	 *
+	 * @param Tribe__Events__Importer__File_Importer|null $instance The default instance that would be used for the type.
+	 * @param Tribe__Events__Importer__File_Reader        $file_reader
+	 *
+	 * @return Tribe__Tickets__CSV_Importer__RSVP_Importer
+	 */
+	public static function instance( $instance, Tribe__Events__Importer__File_Reader $file_reader ) {
```

Comment:
I'd leave it as this is a filter.
The unusual situation here is the default value is `null` but that makes sense.
I see the case where we or a  third party would like to do the following things on that `$instance`:

* create it to handle the case (what happens now)
* trash it and return a new one suited to a very specific task (think of an instance handling WooCommerce tickets with or without recurrence support)
* augment the instance setting properties on it
* decorate the instance using it as a decorator pattern chain (ok, unlikely but I'd cherish the possibility)